package.json

```diff
@@ -8,7 +8,7 @@
   "engines": {
     "vscode": "^1.86.0",
     "npm": "8.1.2",
-    "node": "16.20.2"
+    "node": "20.11.0"
```

Comment:
Please keep it to previous value, we'll update it later

Reply:
Done